parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/ConsumerManager.java

```diff
@@ -65,8 +70,9 @@ ConsumerRecords<K, V> poll(Duration requestedLongPollTimeout) {
         return records;
     }
 
-    protected void updateMetadataCache() {
+    protected void updateCache() {
         metaCache = consumer.groupMetadata();
+        pausedPartitionsCache.set(consumer.paused());
```

Comment:
If you accept the previous change.
```suggestion
        pausedPartitionSize.set(consumer.paused().size());
```